build.assets/robotest/test.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+source $(dirname $0)/utils.sh
+
+# see https://github.com/gravitational/gravity/issues/1735 for explanation of why
+# 6.1.2x+ to 7.0.0 isn't supported.
+if [[ "$(recommended_upgrade_tag_between $(branch 6.1.x) 7.0.0)" != "6.1.20" ]];  then
+  exit 1
+fi
+
+# check a more current 7.0.x has a more current corresponding 6.0.x
+if [[ "$(recommended_upgrade_tag_between $(branch 6.1.x) 7.0.11)" != "6.1.29" ]];  then
```

Comment:
I'm not sure, what is this testing? Do we need to keep these version up-to-date when next 6.1/7.0 are released?

Reply:
Nothing in this file should need to be kept up to date for a couple reasons:

1) In order to avoid etcd mismatches the current algorithm will not recommend from tags/commits more current than the to build.  That means we should never recommend a CI upgrade from 6.1.40 -> 7.0.11.  It will be pinned at 6.1.29 forever.

2) These are unit tests for the code in utils.sh.  If the test cases are out of date, they will never affect what tests are actually run.  I'll add a comment to the file to help clarify what the intention is.

Reply:
Fixed in ec2a27a.